### PR TITLE
Add a banner with the gofundme

### LINKED ISSUE #378: Add GoFundMe to site banner
**Is your feature request related to a problem? Please describe.**
GoFundMe link is currently on landing page but not as visible. We're low on cash and need to fundraise more!

**Describe the solution you'd like**
Find the existing banner template under `templates/common/notification.html`, edit the contents, and import it into the base site template so it shows at the top of the page below the navbar.

**Describe alternatives you've considered**
Any other ideas for fundraising?

**Additional context**
Previous banner:
image

### PR DESCRIPTION
**Pull request by khushic**
## Status
- Done

## GitHub Issues addressed (`#378`)
- This PR closes #378 

## What I did
- Modified the notification template in common/notification.html to include the gofundme link
- Imported the banner into base.html

## Screenshots
- Before
Screen Shot 2021-02-14 at 1 27 24 PM
- After
Screen Shot 2021-02-14 at 1 27 45 PM

## Tests
- Check that the banner shows up and the link takes you to the gofundme link

## Questions/Discussions/Notes
- N/A

## Merging the PR
- Who should merge this PR?
  - The last reviewer to approve can merge it
- Should the commit history be preserved in the base branch, or is it okay to combine all of them into a single commit ("squash-merge")?
  - preserve the history

### CODE REVIEW

File: tcf_core/settings/base.py [hunk 1]
```diff
@@ -7,7 +7,10 @@
 import environ
 
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
-BASE_DIR = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+BASE_DIR = os.path.dirname(
```
Comment (j-alicia-long) [inline tcf_core/settings/base.py:10]: Is this the autoformatting again? might wanna discard these changes hmm

File: tcf_website/urls.py [hunk 2]
```diff
@@ -11,7 +11,10 @@
     path('terms', views.terms, name='terms'),
     path('browse', views.browse, name='browse'),
     path('department/<int:dept_id>', views.department, name='department'),
-    path('course/<int:course_id>', views.course_view_legacy, name='course_legacy'),
```
Comment (j-alicia-long) [inline tcf_website/urls.py:14]: same here

**Review by j-alicia-long — COMMENTED** [review]
Looks good to me! Once you resolve those merge conflicts I can approve it. 

Btw D&M ppl were thinking abt starting a Venmo fundraiser if we wanna look into including that?

File: tcf_core/settings/base.py (hunk 1 above)
Comment (khushic) [reply tcf_core/settings/base.py:10]: I think they came up right after I opened my new branch so I think it is autoformatting I'll discard them and not commit them for future issues/features

**Comment by khushic** [comment]
Yup once they have the link for that I can add a second button with labels for gofundme vs venmo

**Comment by khushic** [comment]
It did something weird when I tried to merge dev into the branch let me know if there were any issues with it

**Review by j-alicia-long — APPROVED** [review]

**Pull request merged**